pocs/images.py

```diff
@@ -102,8 +102,9 @@ def wcs_file(self, filename):
 
                 self.wcs = w
                 self._wcs_file = filename
+                self.logger.debug("WCS loaded from image")
             except Exception:
-                self.logger.debug("Can't get WCS from FITS file (try solve_field)")
+                pass
```

Comment:
This is an odd setter, right? It is a no-op if the file doesn't exist, isn't a FITS file or if the file doesn't contain the WCS data.

Reply:
Yes. This is actually an odd file altogether with how it works although it is useful. Josh did the original and then I cleaned it up a little but I think my cleanup (including things like this setter) made it oddly formed.